src/helm/benchmark/metrics/metric.py

```diff
@@ -352,3 +333,19 @@ def add_context(stat: Stat, context: MetricContext) -> Stat:
     return Stat(
         replace(stat.name, split=context.split, sub_split=context.sub_split, perturbation=context.perturbation)
     ).merge(stat)
+
+
+def get_num_train_trials(request_states: List[RequestState]) -> int:
```

Comment:
There appears to be no method calling this? Is it left over from a previous iteration?